docs/manual/exporting.md

```diff
@@ -4,13 +4,13 @@ title: Exporting your work
 sidebar_label: Exporting
 ---
 
-## Overview {#overview}
+## Overview \{#overview}
```

Comment:
Any idea why escaping would be needed here but not in other files where the heading ids are also made explicit in the source? For instance, you did not escape them in `docs/manual/grelfunctions.md` but that page seems to still compile fine, no?

Reply:
Yes, went through the documentation of  Docusaurus 3.0 where there is a command to check what will lead error if we upgrade where it shows the problems to the file and I rectified it according to that only. Where in exporting.md I had to do this because while compiling it throws error of react child class can't be rendered, it means it was taking it as a react class so added escape here and it worked.

Reply:
I rectified it this time. Thank you and sorry for this feedback.

Reply:
>so added escape here and it worked.
The `{#overview}` markdown is meant to set a specific anchor name (see https://docusaurus.io/docs/markdown-features/toc#heading-ids). So when you say "it worked" do you mean that there was no longer an error, or did you check that the anchor still works?

Reply:
In any situation where we are explicitly setting an anchor on a heading that would be the same as the automatically generated one, we could remove the explicit setting (see https://docusaurus.io/docs/markdown-features/toc#heading-ids)

Reply:
The intention of adding those anchors was to ensure their stability (to avoid breaking links) when we reword the heading titles. When those headings are written in the source, you can still change them but at least you are doing so consciously.

We did that when preparing the documentation for translation because it was recommended in Docusaurus' docs, I think. Of course, there has not been much progress on the translation front, but we got some feedback that translating the docs would indeed be very useful to some communities (in fact, more useful than translating the UI of the tool, because that makes it harder to relate to tutorials in other languages, or to ask for support in English).

Reply:
![image](https://github.com/OpenRefine/openrefine.org/assets/126223325/064ee224-8d93-4946-8933-e80b0bd57a16)
I found this error now... Can you help me understand is there any react child?

Reply:
Never mind I found it and I guess this time it will work.